conductr_cli/constants.py

```diff
@@ -14,6 +14,10 @@
                                          '{}/settings.conf'.format(DEFAULT_CLI_SETTINGS_DIR))
 DEFAULT_CUSTOM_PLUGINS_DIR = os.getenv('CONDUCTR_CUSTOM_PLUGINS_DIR',
                                        '{}/plugins'.format(DEFAULT_CLI_SETTINGS_DIR))
+DEFAULT_SANDBOX_IMAGE_DIR = os.path.abspath(os.getenv('CONDUCTR_SANDBOX_IMAGE_DIR',
+                                                      '{}/images'.format(DEFAULT_CLI_SETTINGS_DIR)))
+DEFAULT_SANDBOX_INTERFACE = os.getenv('CONDUCTR_SANDBOX_INTERFACE', 'lo0')
```

Comment:
Should we detect our OS here and default `lo` for Linux, `lo0` for OS X at least? This explicit setting of a loopback interface may still be useful, even with our forthcoming auto-detection.